srml/staking/src/phragmen.rs

```diff
@@ -188,6 +202,17 @@ pub fn elect<T: Trait + 'static, FR, FN, FV, FS>(
         if candidates.len() > minimum_validator_count {
             // if we don't have enough candidates, just choose all that have some vote.
             elected_candidates = candidates;
+			// `Exposure.others` still needs an update
```

Comment:
@gavofyork this has room for debate. Current descision: 
- With _code yellow_ where we have some votes but less than needed: exposure is updated. 
- With _code red_ where we have some votes and less than Minimum: exposure is NOT updated. This is because technically in this scenario votes don't really matter. Though, note that `Stakers<T>` is cleaned outside of the functions so they will actually also NOT have even the previous era's exposure. An alternative is to keep the exposure that they used to have if this _code red_ happened.

Reply:
that's fine for now. it's something we should put to the w3f researchers though.

Reply:
Will post a detailed description there.